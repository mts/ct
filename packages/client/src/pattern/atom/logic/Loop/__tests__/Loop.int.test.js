import { defaultRender } from './Loop.int.render'

describe('<Loop />', () => {
  describe('Snaphot', () => {
    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
