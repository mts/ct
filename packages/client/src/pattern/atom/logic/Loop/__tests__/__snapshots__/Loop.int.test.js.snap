// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loop /> Snaphot must match defaultRender 1`] = `
<div
  className=""
>
  <div>
    1
  </div>
  <div>
    2
  </div>
  <div>
    Foo
  </div>
  <div>
    4
  </div>
  <div>
    Bar
  </div>
  <div>
    Foo
  </div>
  <div>
    7
  </div>
  <div>
    8
  </div>
  <div>
    Foo
  </div>
  <div>
    Bar
  </div>
  <div>
    11
  </div>
  <div>
    Foo
  </div>
  <div>
    13
  </div>
  <div>
    14
  </div>
  <div>
    FooBar
  </div>
  <div>
    16
  </div>
  <div>
    17
  </div>
  <div>
    Foo
  </div>
  <div>
    19
  </div>
  <div>
    Bar
  </div>
  <div>
    Foo
  </div>
  <div>
    22
  </div>
  <div>
    23
  </div>
  <div>
    Foo
  </div>
  <div>
    Bar
  </div>
  <div>
    26
  </div>
  <div>
    Foo
  </div>
  <div>
    28
  </div>
  <div>
    29
  </div>
  <div>
    FooBar
  </div>
  <div>
    31
  </div>
  <div>
    32
  </div>
  <div>
    Foo
  </div>
  <div>
    34
  </div>
  <div>
    Bar
  </div>
  <div>
    Foo
  </div>
  <div>
    37
  </div>
  <div>
    38
  </div>
  <div>
    Foo
  </div>
  <div>
    Bar
  </div>
  <div>
    41
  </div>
  <div>
    Foo
  </div>
  <div>
    43
  </div>
  <div>
    44
  </div>
  <div>
    FooBar
  </div>
  <div>
    46
  </div>
  <div>
    47
  </div>
  <div>
    Foo
  </div>
  <div>
    49
  </div>
  <div>
    Bar
  </div>
  <div>
    Foo
  </div>
  <div>
    52
  </div>
  <div>
    53
  </div>
  <div>
    Foo
  </div>
  <div>
    Bar
  </div>
  <div>
    56
  </div>
  <div>
    Foo
  </div>
  <div>
    58
  </div>
  <div>
    59
  </div>
  <div>
    FooBar
  </div>
  <div>
    61
  </div>
  <div>
    62
  </div>
  <div>
    Foo
  </div>
  <div>
    64
  </div>
  <div>
    Bar
  </div>
  <div>
    Foo
  </div>
  <div>
    67
  </div>
  <div>
    68
  </div>
  <div>
    Foo
  </div>
  <div>
    Bar
  </div>
  <div>
    71
  </div>
  <div>
    Foo
  </div>
  <div>
    73
  </div>
  <div>
    74
  </div>
  <div>
    FooBar
  </div>
  <div>
    76
  </div>
  <div>
    77
  </div>
  <div>
    Foo
  </div>
  <div>
    79
  </div>
  <div>
    Bar
  </div>
  <div>
    Foo
  </div>
  <div>
    82
  </div>
  <div>
    83
  </div>
  <div>
    Foo
  </div>
  <div>
    Bar
  </div>
  <div>
    86
  </div>
  <div>
    Foo
  </div>
  <div>
    88
  </div>
  <div>
    89
  </div>
  <div>
    FooBar
  </div>
  <div>
    91
  </div>
  <div>
    92
  </div>
  <div>
    Foo
  </div>
  <div>
    94
  </div>
  <div>
    Bar
  </div>
  <div>
    Foo
  </div>
  <div>
    97
  </div>
  <div>
    98
  </div>
  <div>
    Foo
  </div>
  <div>
    Bar
  </div>
</div>
`;
